src/Microsoft.AspNetCore.StaticFiles/CacheProfile.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.StaticFiles
+{
+    /// <summary>
+    /// Defines settings used for caching responses for the <see cref="StaticFileMiddleware"/>.
+    /// </summary>
+    public class CacheProfile
+    {
+        /// <summary>
+        /// Gets or sets the duration in seconds for which the response is cached.
+        /// Sets the "max-age" in the "Cache-control" header in the 
+        /// <see cref="Microsoft.AspNetCore.Http.HttpContext.Response" />.
+        /// Defaults to 10 minutes
+        /// </summary>
+        public int Duration { get; set; } = 600;
```

Comment:
TimeSpan is better for helping users understand the units. Otherwise you'd have to name the property DurationInSeconds (please no).